network.go

```diff
@@ -390,6 +390,50 @@ func createNetworkEndpoints(numOfEndpoints int) (endpoints []Endpoint, err error
 	return endpoints, nil
 }
 
+func getIfacesFromNetNs(networkNSPath string) ([]net.Interface, error) {
+	var ifaces []net.Interface
+	var err error
+
+	if networkNSPath == "" {
+		return []net.Interface{}, fmt.Errorf("Network namespace path cannot be empty")
+	}
+
+	err = doNetNS(networkNSPath, func(_ ns.NetNS) error {
+		ifaces, err = net.Interfaces()
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+	if err != nil {
+		return []net.Interface{}, err
+	}
+
+	return ifaces, err
+}
+
+func getIPNetFromEndpoint(endpoint Endpoint, netIfaces []net.Interface) (net.IPNet, error) {
```

Comment:
No we cannot do that but I don't think we want to. We have several endpoints in virtcontainers, describing the veth end point inside the netns. For each of these endpoints, there is only one ipAddress, and we can give hyperstart as many as he wants. We don't have the framework to support this in virtcontainers right now, and I am wondering about the use cases.